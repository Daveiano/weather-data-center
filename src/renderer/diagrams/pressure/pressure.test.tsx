import React from 'react'
import { render, screen } from '@testing-library/react'

import data from "../../../../tests/data/scaling-input.json";

import PressureBase from "./pressure-base";

test('pressure diagram with 11-day data', async () => {
  const { container } = render(<PressureBase title="Pressure" height="300px" data={data} property="pressure" />);

  expect(screen.getByTestId('pressure-diagram')).toHaveTextContent("Pressure");

  expect(container).toMatchSnapshot();
});