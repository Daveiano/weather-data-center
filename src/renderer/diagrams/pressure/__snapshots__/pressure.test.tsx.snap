// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pressure diagram with 11-day data 1`] = `
<div>
  <div
    class="bx--col"
  >
    <div
      class="bx--tile"
    >
      <div
        data-testid="pressure-diagram"
      >
        <h3>
          <svg
            aria-hidden="true"
            fill="currentColor"
            focusable="false"
            height="32"
            preserveAspectRatio="xMidYMid meet"
            viewBox="0 0 32 32"
            width="32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M17.5053,16l8.1591-7.2529A1,1,0,0,0,25,7H22V2H20V9h2.37L16,14.6621,9.63,9H12V2H10V7H7a1,1,0,0,0-.6646,1.7471L14.4945,16,6.3353,23.2529A1,1,0,0,0,7,25h3v5h2V23H9.63L16,17.3379,22.37,23H20v7h2V25h3a1,1,0,0,0,.6645-1.7471Z"
            />
          </svg>
          Pressure
        </h3>
        <div
          class="diagram"
          style="height: 300px;"
        >
          <div
            style="width: 100%; height: 100%;"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
